docs/source/user-guide/vision/tutorials/plot_custom_task_tutorial.py

```diff
@@ -154,7 +154,7 @@ def load_or_download(cls, root: Path, train: bool) -> 'CocoSegmentDataset':
                     extract_root=str(root),
                     md5=md5
                 )
-            
+
```

Comment:
why

Reply:
Because useless spaces are useless, of course.